src/backend/distributed/citus.control

```diff
@@ -1,6 +1,6 @@
 # Citus extension
 comment = 'Citus distributed database'
-default_version = '10.2-2'
+default_version = '10.2-3'
```

Comment:
We expect to see upgrade and downgrade tests in `multi_extension` whenever we bump the schema version of our extensions.

This was missing from this PR, however we fixed it in all necessary branches.
 
#5335 adds the missing tests for `master` and all release branches >= 11
#5421 will add the tests for